mirth.rb

```diff
@@ -35,25 +33,22 @@ class DailyData < ActiveRecord::Base; end
         <p><button>Submit daily data</button></p>
       </form>
     STR
-    response.finish
-  }
+ 
+    render(html: response_body.html_safe)
+  end
 
-  post '/add/data', to: -> environment {
-    request = Rack::Request.new(environment)
-    response = Rack::Response.new
-    params = request.params
+  def add_data
     DailyData.create(date: params['date'], step_count: params["step_count"], notes: params["notes"])
-    response.redirect('/show/data', 303)
-    response.finish
-  }
-
-  match '*path', via: :all, to: -> environment {
-    request = Rack::Request.new(environment)
-    response = Rack::Response.new  
-    response.content_type = "text/plain"
-    response.write("✅ Received a #{request.request_method} request to #{request.path}!")
-    response.finish
-  }
+    redirect_to("/show/data")
```

Comment:
The default redirect status is 302 when there isn't a status specified. 

```
127.0.0.1 - - [23/Feb/2021:13:14:42 -0500] "POST /add/data HTTP/1.1" 302 - 0.2281
```

I figured we keep it simple here, but we can also `redirect_to("/show/data", status: 303)`

Reply:
I agree it’s good to keep it simple, but this is a good opportunity to use another nice feature of Action Controller. Check out [the documentation for the `:status` option](https://guides.rubyonrails.org/layouts_and_rendering.html#the-status-option) (supported by both `render` and `redirect_to`) for a new and more readable way to set the status code.

Reply:
Added `render(status: :see_other)`

Reply:
As you can see if you run it 👀, this isn’t right.